src/android/SpectrumManager.java

```diff
@@ -80,6 +80,12 @@ private void transcodeImage(String path, int size, CallbackContext callbackConte
                     callbackContext.error("Failed to compress image");
                     return;
                 }
+            } else {
+                Log.d("Warning","Cannot compress image on Android version < 11");
```

Comment:
```suggestion
                Log.d("Warning","Cannot compress image on Android version < 11");
```

can we use `callbackContext.error` here